webgpu.h

```diff
@@ -448,7 +425,7 @@ typedef enum WGPUSType {
     WGPUSType_SurfaceDescriptorFromWaylandSurface = 0x00000008,
     WGPUSType_SurfaceDescriptorFromAndroidNativeWindow = 0x00000009,
     WGPUSType_SurfaceDescriptorFromXcbWindow = 0x0000000A,
-    WGPUSType_RenderPassDescriptorMaxDrawCount = 0x0000000F,
+    WGPUSType_RenderPassDescriptorMaxDrawCount = 0x0000000B,
```

Comment:
I think the YAML could specify a value for this so that it doesn't get renumbered? (We're going to need to renumber all these anyway later but it's convenient to minimize the differences for now)